src/mentionui.js

```diff
@@ -273,6 +273,26 @@ export default class MentionUI extends Plugin {
 
 			const { feedText, marker } = matched;
 
+			const matchedTextLength = marker.length + feedText.length;
+
+			// create marker range
+			const start = selection.focus.getShiftedBy( -matchedTextLength );
+			const end = selection.focus.getShiftedBy( -feedText.length );
+
+			const markerRange = editor.model.createRange( start, end );
+
+			let mentionMarker;
+
+			if ( editor.model.markers.has( 'mention' ) ) {
+				mentionMarker = editor.model.markers.get( 'mention' );
+			} else {
+				mentionMarker = editor.model.change( writer => writer.addMarker( 'mention', {
```

Comment:
Do we destroy this marker? Or we keep it forever because it's just one marker so it does not make sense to destroy it?

Reply:
I think both:
1. There's always 1 `mentions` marker to track marker in the editing area for positioning purposes (ps.: I wanted to name that variable `markerMarker` but it would be too dead joke :D )
2. I remove this marker when panel is closed (`MentionUI#_hidePanelAndRemoveMarker()`)

Reply:
OK, I see it below in `hidePanelAndRemoveMarker`. Nvmnd.